RecipesBase/src/RecipesBase.jl

```diff
@@ -332,18 +330,13 @@ end
 ```
 """
 macro series(expr::Expr)
-    esc(
-        quote
-            let plotattributes = copy(plotattributes)
-                args = $expr
-                push!(
-                    series_list,
-                    RecipesBase.RecipeData(plotattributes, RecipesBase.wrap_tuple(args)),
-                )
-                nothing
-            end
-        end,
-    )
+    esc(quote
+        let plotattributes = copy(plotattributes)
+            args = $expr
+            push!(series_list, $RecipeData(plotattributes, $wrap_tuple(args)))
+            nothing
+        end
+    end)
```

Comment:
```suggestion
    quote
        let plotattributes = copy(plotattributes)
            args = $expr
            push!(series_list, $RecipeData(plotattributes, $wrap_tuple(args)))
            nothing
        end
    end |> esc
```

Minor stylistic enhancement.